streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/CogroupedKStream.scala

```diff
@@ -82,6 +82,16 @@ class CogroupedKStream[KIn, VOut](val inner: CogroupedKStreamJ[KIn, VOut]) {
   def windowedBy[W <: Window](windows: Windows[W]): TimeWindowedCogroupedKStream[KIn, VOut] =
     new TimeWindowedCogroupedKStream(inner.windowedBy(windows))
 
+  /**
+   * Create a new [[TimeWindowedCogroupedKStream]] instance that can be used to perform windowed aggregations.
```

Comment:
nit: `perform sliding windowed aggregations` instead of `perform windowed aggregations`